frontend/wallet/schema.graphql

```diff
@@ -136,7 +119,13 @@ input CreatePaymentInput {
   memo: String
 }
 
-input PaymentFilterInput {
+input CreateDelegationInput {
+  from: String!,
+  to: String,
+  fee: String!,
```

Comment:
You can still technically add memos for `delegation`. Can we add a `memo` field to delegation?

Reply:
ok I guess so